versions.cfg

```diff
@@ -25,13 +24,17 @@ Sphinx = >=1.3.1
 docutils = >=0.12
 Pygments = >=2.0.2
 
+[versions:python27]
+configparser = 4.0.2
+zipp = 1.2
+
 [repo]
 rw = git@github.com:bluedynamics
 ro = https://github.com/bluedynamics
 
 [sources]
 # node                 = git ${repo:ro}/node.git pushurl=${repo:rw}/node.git
-node.ext.ldap        = git ${repo:ro}/node.ext.ldap.git pushurl=${repo:rw}/node.ext.ldap.git
+node.ext.ldap        = git ${repo:ro}/node.ext.ldap.git pushurl=${repo:rw}/node.ext.ldap.git branch=external-groups
```

Comment:
this checkout can be commented and pointed to master again. related code already released on pypi.

Reply:
done